shared/constants/network.ts

```diff
@@ -416,10 +420,10 @@ export const ETHERSCAN_SUPPORTED_NETWORKS = {
     }`,
     networkId: CHAIN_ID_TO_NETWORK_ID_MAP[CHAIN_IDS.SEPOLIA],
   },
-  [CHAIN_IDS.LINEA_TESTNET]: {
-    domain: 'linea.build',
-    subdomain: 'explorer.goerli',
-    networkId: CHAIN_ID_TO_NETWORK_ID_MAP[CHAIN_IDS.LINEA_TESTNET],
+  [CHAIN_IDS.LINEA_GOERLI]: {
+    domain: 'lineascan.build',
```

Comment:
I understand that there is a plan to switch to this block tracker, but I don't think we should make this change until the site is live